lib/jsdom/living/register-elements.js

```diff
@@ -283,6 +283,38 @@ const mappings = {
   HTMLVideoElement: {
     file: require("./generated/HTMLVideoElement.js"),
     tags: ["video"]
+  },
+  SVGCircleElement: {
+    file: require("./generated/SVGCircleElement.js"),
+    tags: ["circle"]
```

Comment:
Hmm. We need tests that this will not cause SVGCircleElement to pop up: `<html><circle>x</circle>`. Instead it needs to be inside a SVG tag.

I doubt that just adding them to register-elements.js will work.